### DISCUSSION TITLE
FEAT: Create 'default' Nash equilibrium methods

### DISCUSSION THREAD

**Discussion by tturocy** [gambitproject/gambit#873] [Ideas]
The graphical interface has long offered some generic methods for computing one, some, or all Nash equilibria, where available.

Similarly, for many types of problem, `scipy` offers generic routines which hide the selection of, e.g. a specific algorithm.  See, for example, function minimisation or ODE solving.

It would be useful to offer something similar in `pygambit`.

In doing so it will be worth giving some thought as to how best to keep this in sync between the graphical interface and `pygambit`.  At the moment the graphical interface makes calls out to the command-line tools, but perhaps this should be reviewed - which would allow for having the logic for EC methods to be maintained in a single location.

**Comment by rahulsavani** [comment]
@tturocy: Should this become a discussion?
